docsteady/config.py

```diff
@@ -93,6 +93,8 @@ class Config:
     TEST_STATUS_COUNT = Counter()
     REQ_PER_DOC = dict()
 
+    ExecutionError = False
```

Comment:
`EXECUTION_ERRORED` is more appropriate as a constant name and is more literate when reading the code. `ExecutionError` is in the form of a class name